ios/FluentUI.Demo/FluentUI.Demo/Demos/PopupMenuDemoController.swift

```diff
@@ -129,4 +130,32 @@ class PopupMenuDemoController: DemoController {
 
         present(controller, animated: true)
     }
+
+    @objc private func showCustomColorsButtonTapped(sender: UIButton) {
+        let controller = PopupMenuController(sourceView: sender, sourceRect: sender.bounds, presentationDirection: .down)
+
+        let items = [
+            PopupMenuItem(image: UIImage(named: "agenda-24x24"), title: "Agenda", isSelected: calendarLayout == .agenda, onSelected: { self.calendarLayout = .agenda }),
+            PopupMenuItem(image: UIImage(named: "day-view-24x24"), title: "Day", isSelected: calendarLayout == .day, onSelected: { self.calendarLayout = .day }),
+            PopupMenuItem(image: UIImage(named: "3-day-view-24x24"), title: "3-Day", isEnabled: false, isSelected: calendarLayout == .threeDay, onSelected: { self.calendarLayout = .threeDay })
+        ]
+
+        let menuBackgroundColor = UIColor(red: 0.2, green: 0.2, blue: 0.2, alpha: 1.0)
+
+        for item in items {
+            item.titleColor = .white
+            item.titleSelectedColor = .white
+            item.imageSelectedColor = .white
+            item.accessoryCheckmarkColor = .white
+            item.backgroundColor = menuBackgroundColor
+        }
+
+        controller.addItems(items)
+
+        controller.backgroundColor = menuBackgroundColor
+        controller.resizingHandleViewBackgroundColor = menuBackgroundColor
+        controller.separatorColor = UIColor(red: 0.3333333333, green: 0.3333333333, blue: 0.3333333333, alpha: 1.0)
```

Comment:
Any reason not to use [opaqueSeparatorColor](https://developer.apple.com/documentation/uikit/uicolor/3173133-opaqueseparatorcolor?language=objc)?

Reply:
(If Fluent wants a different color, that's fine, but this is in the test code and is hardcoded)

Reply:
i don't care either way. i was just copying from PDFkit's codebase.

Reply:
opaqueSeparatorColor is only iOS 13 so i probably use just lightGray or something else